pkg/operator/starter.go

```diff
@@ -242,7 +244,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		kubeInformersForNamespaces.InformersFor(operatorclient.TargetNamespace).Core().V1().Namespaces().Lister(),
 		configInformers.Config().V1().Infrastructures().Lister(),
 		operatorClient,
-		etcdClient)
+		cachedMemberClient)
```

Comment:
just going through some other controllers that could be cached:
* NewEtcdMembersController <- that's just informatory, so doesn't need consistent information
* NewDefragController <- that's also not mission critical to be consistent, it's also quite shoddy in how often it calls the member listing


anything else that springs to your mind?

Reply:
i also had another run over CEO ... i cant find anything that will need `memberList()` periodically

i think if we utilise this in the `clientPool` it will be best however, wdyt ?

Reply:
do you want to update the members and defrag controller with this as well?

Reply:
👍🏽